sdk/resourcemanager/containerinstance/armcontainerinstance/autorest.md

```diff
@@ -5,8 +5,8 @@
 ``` yaml
 azure-arm: true
 require:
-- https://github.com/Azure/azure-rest-api-specs/blob/7a2ac91de424f271cf91cc8009f3fe9ee8249086/specification/containerinstance/resource-manager/readme.md
-- https://github.com/Azure/azure-rest-api-specs/blob/7a2ac91de424f271cf91cc8009f3fe9ee8249086/specification/containerinstance/resource-manager/readme.go.md
+- https://github.com/Azure/azure-rest-api-specs/blob/b23f35dfc3ceee0a84c1380b02bcfdfa7f23049c/specification/containerinstance/resource-manager/readme.md
+- https://github.com/Azure/azure-rest-api-specs/blob/b23f35dfc3ceee0a84c1380b02bcfdfa7f23049c/specification/containerinstance/resource-manager/readme.go.md
 license-header: MICROSOFT_MIT_NO_VERSION
-module-version: 1.0.0
+module-version: 2.0.0
```

Comment:
```suggestion
module-version: 1.1.0
```

Reply:
@RickWinter I think we have the conclusion that any client change should release a major version. Could you help to sign off?